cineast-runtime/src/main/java/org/vitrivr/cineast/standalone/run/GenericExtractionItemHandler.java

```diff
@@ -128,6 +134,17 @@ public GenericExtractionItemHandler(ExtractionContainerProvider pathProvider, Ex
     }
     handlers.put(MediaType.AUDIO, new ImmutablePair<>(FFMpegAudioDecoder::new, () -> new ConstantLengthAudioSegmenter(context)));
     handlers.put(MediaType.VIDEO, new ImmutablePair<>(FFMpegVideoDecoder::new, () -> new VideoHistogramSegmenter(context)));
+    handlers.put(MediaType.TEXTUREMODEL3D, new ImmutablePair<>(ModularTextureModelDecoder::new, () -> new PassthroughSegmenter<Model>() {
+      @Override
+      protected SegmentContainer getSegmentFromContent(Model content) {
+        return new TextureModel3DSegment(content);
+      }
+
+      @Override
+      public Set<MediaType> getMediaTypes() {
+        return Sets.newHashSet(MediaType.values());
```

Comment:
I don't think this is correct, right? The TEXTUREMODEL3D segmenter can't segment all media types? I assume this is due to the fact that the spec for the Model3D segmenter is also wrong right below this change